projects/plugins/jetpack/extensions/blocks/subscriptions/block.json

```diff
@@ -108,6 +108,10 @@
 		"successMessage": {
 			"type": "string",
 			"default": "Success! An email was just sent to confirm your subscription. Please find the email now and click 'Confirm Follow' to start subscribing."
+		},
+		"source": {
```

Comment:
Thinking if it would make sense to prefix with `__source` to clarify it's sort of internal attribute?